playbooks/vars/main.yml

```diff
@@ -59,7 +59,7 @@ maas_fqdn_extension: ""
 # The entity name is the inventory hostname and fqdn extension combined.
 #
 # Set the hostname for use with the api checks
-maas_entity_name: "{{ inventory_hostname }}{{ maas_fqdn_extension }}"
+maas_entity_name: "{{ ansible_hostname }}{{ maas_fqdn_extension }}"
```

Comment:
Can we get more info on this change here?

Reply:
From the commit message:
```
The curently used inventory_hostname does not match the actual
entitiy names. Tied to RE-645 as the problem was discovered while
working on phobos deploy verification job.```

Reply:
OK ... I guess my concern is that this code is being used in prod now, so it's a little unclear why this isn't a problem for other deployments.

Reply:
This change make a bit more sense now that I understand that deployers don't always set the inventory_hostname to match their system's hostname (ansible_hostname).